src/cmds/qstat.c

```diff
@@ -294,6 +309,47 @@ states(char *string, char *q, char *r, char *h, char *w, char *t, char *e, int l
 	}
 }
 
+/**
+ * @brief
+ *	Prepend escape character with special characters such as line feed and form feed
+ *
+ * @param[in]   val - string which needs to be formatted
+ *
+ * @return string
+ * @retval NULL     - if memory allocation failed.
+ * @retval string   - formatted string.
+ *
+ */
+char *
+escape_feed_chars(char *val) {
```

Comment:
@arungrover, yes, encode in PBS has a different meaning.. but i am not satisfied yet that we are escaping the feed characters - we are replacing "\n" with two slashes and a 'n'..